qp_shotgun/humann2/humann2.py

```diff
@@ -186,13 +186,24 @@ def humann2(qclient, job_id, parameters, out_dir):
                      '-o {0}/pathcoverage_relab.biom').format(out_dir))
     commands.append(('humann2_renorm_table -i {0}/pathabundance.biom -u relab '
                      '-o {0}/pathabundance_relab.biom').format(out_dir))
-    msg = "Step 5 of 5: Re-normalizing tables (%d/3)"
+    msg = "Step 5 of 6: Re-normalizing tables (%d/3)"
     success, msg = _run_commands(qclient, job_id, commands, msg)
     if not success:
         return False, None, msg
 
-    # Generating 6 artifacts, separation is important for analysis
+    # Step 6 stratifiying re-normalized tables
+    commands = []
     pb = partial(join, out_dir)
+    cmd = "humann2_split_stratified_table --input %s --output %s"
+    commands.append(cmd % (pb(out_dir, 'genefamilies_cpm.biom'), out_dir))
+    commands.append(cmd % (pb(out_dir, 'pathcoverage_relab.biom'), out_dir))
+    commands.append(cmd % (pb(out_dir, 'pathabundance_relab.biom'), out_dir))
+    msg = "Step 6 of 6: Stratifiying re-normalizing tables (%d/3)"
+    success, msg = _run_commands(qclient, job_id, commands, msg)
+    if not success:
+        return False, None, msg
+
+    # Generating 6 artifacts, separation is important for analysis
     ainfo = [
         ArtifactInfo('Gene family table', 'BIOM',
```

Comment:
Each of these "ArtifactInfo" should be listed in the outputs dictionary of the __init__.py since that dictionary should have 1 entry per Artifact generated.